src/dataHelpers.jl

```diff
@@ -12,7 +12,7 @@ function geocmean(x)
 end
 
 const cellTypes = [:ncMO, :cMO, :NKs, :Neu, :EO]
-const murineIgG = [:IgG1, :IgG2a, :IgG2b, :IgG3]
+const murineIgG = [:IgG1, :IgG2a, :IgG2b, :IgG3, :IgG2bFucose]
```

Comment:
You probably want to test if this will break other functions and/or figures

Reply:
I searched "murineIgG" in this repository and there were very few places that called this constant. However, I received this build error message, and I got stuck here:
Test that more refined brute force improves match.: Error During Test at /home/jenkins/workspace/meyer-lab_FcgR.jl_PR-232-merge/test/testTranslation.jl:2
  Got exception outside of a @test
  TaskFailedException:
  AssertionError: ni == length(IgGC)
I couldn't find the above assertion in this repository

Reply:
I see. The translation assumes both mouse and human have four subtypes of IgG, if you add one more to `murineIgG` it will break this assumption.